packages/dara-components/dara/components/graphs/components/base_graph_component.py

```diff
@@ -42,7 +42,7 @@ class BaseGraphComponent(StyledComponentInstance):
         by default will also update the `causal_graph` provided, if it's a plain Variable instance
     :param tooltip_size: Optional parameter to force the tooltips to use a particular font size
     :param verbose_descriptions: Optional flag to show verbose descriptions in the editor frame
-    :param zoom_thresholds: Optional user-defined zoom thresholds. See `ZoomThresholds` for more details.
+    :param zoom_thresholds: Optional user-defined zoom thresholds. See ![ZoomThresholds](../definitions/#zoomthresholds) for more details.
```

Comment:
@krzysztof-causalens I remember we talked about this, but can't remember what was decided. Are we going to add reference links here or did we decide on not to because it doesn't work for IDE hover?

Reply:
Not sure if we decided on anything. It's awkward this doesn't work inside an IDE but probably worth it to make docs work better